cli/src/cmd/forge/coverage.rs

```diff
@@ -105,6 +105,8 @@ impl CoverageArgs {
 
             // Disable the optimizer for more accurate source maps
             project.solc_config.settings.optimizer.disable();
+            project.solc_config.settings.optimizer.runs = Some(0);
```

Comment:
what if we set this to `None`? does this have any effect?

Reply:
The codehash changes but it still has the desired effect of isolating the coverage runs from `optimzer_runs` setting. I.e. if I then change my optimizer settings they have no effect on codehash (because it gets overwritten before run).

I'm happy with either approach, `None` may be more cleaner so if that works that's cool